README.md

```diff
@@ -1 +1,404 @@
 # api-intro-with-ajax
+
+## Learning Objectives
+
+- Describe what an API is, and why we might use one.
+- Understand the format of API data
+- Explain the common role of JSON on the web.
+- Use jQuery $.ajax() method to make GET requests for data.
+- Use jQuery's 'promise-like' methods to handle AJAX responses.
+- Render new HTML content using data loaded from an Ajax request.
+- Perform POST, PUT, and DELETE requests to an API to modify data.
+
+
+## Framing (5 mins)
+
+**What is an API?**
+>Basically, an API is a service that provides raw data for public use.
+
+API stands for "Application Program Interface", and technically applies to all of software design. However, since the explosion of information technology, the term now commonly refers to web URLs that can be accessed for raw data.
+
+APIs publish data for public use. As third-party software developers, we can access an organization's API and use their data within our own applications.
+
```

Comment:
Can also mention that we've already encountered an API in Active Record, if you want to provide an example of one that is not a "web URL that can be accessed for raw data."